CHANGES_NEXT_RELEASE

```diff
@@ -3,4 +3,5 @@
 - Fix: proper 500 Internal Server Error response with descriptive text when "sort fail" at DB occur (#2752)
 - Add: metrics REST API (#2750)
 - Add: new CLI '-disableMetrics' to disable metrics (#2750)
+- Add: timeout of 6 seconds (hardcoded for now) for all incoming REST requests (#2762)
```

Comment:
As this is not a definitive solution, my suggestion is change: (#2762) -> (temporal fix for #2762) or something similar.